testlib/src/main/java/com/diffplug/spotless/ResourceHarness.java

```diff
@@ -50,36 +45,12 @@
 	 * and {@link #newFile(String)} apis, we can guarantee there
 	 * will be no symlink problems.
 	 */
-	@Rule
-	public TemporaryFolder folderDontUseDirectly = new TemporaryFolder();
-
-	/** Log nontruncated diff in case of a comparison failure to ease test development.*/
-	@Rule
-	public TestWatcher logComparisonFailureDiff = new TestWatcher() {
```

Comment:
I wasn't quite sure if this `TestWatcher` is useful and removed it. Please let me know if it would be valuable to have some JUnit 5 replacement that prints the diff

Reply:
I'm not sure how important this is either, but if it turns out to be important we can put work into restoring something like it later.